docs/intro.md

```diff
@@ -59,8 +59,8 @@ backend:
 media_folder: "img/uploads" # Folder where user uploaded files should go
 
 collections: # A list of collections the CMS should be able to edit
-  - name: "post" # Used in routes, ie.: /admin/collections/:slug/edit
-    label: "Post" # Used in the UI, ie.: "New Post"
+  - name: "post" # Used in routes, i.e.,: /admin/collections/:slug/edit
+    label: "Post" # Used in the UI, i.e.,: "New Post"
```

Comment:
Comma and colon together seems a bit much. Maybe just the comma? Also, now that I'm looking at it, `e.g.` seems more appropriate than `i.e.`